tests/ga/test_extension.py

```diff
@@ -769,6 +769,108 @@ def test_extensions_deleted(self, *args):
         self._assert_handler_status(protocol.report_vm_status, "Ready", 1, "1.0.1")
         self._assert_ext_status(protocol.report_ext_status, "success", 0)
 
+    @patch('subprocess.Popen.poll')
+    def test_install_failure(self, patch_poll, *args):
```

Comment:
A method comment would make it clearer what you are actually trying to assert in this case.  The comment below makes that clearer, but a little verbiage would help.

Reply:
Added some comments